cfn-resources/search-deployment/test/inputs_1_create.template.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Profile": "default",
+  "ProjectId": "",
+  "ClusterName": "",
+  "Specs": [
+    {
+      "InstanceSize": "S30_HIGHCPU_NVME",
+      "NodeCount": "3"
```

Comment:
Defined NodeCount as a string (is actually a string) as a work around for a failure faced in contract tests. Planning to raise an issue to CFN.

Reply:
ok, curious to know more about this

Reply:
I see `"type": "number"` configured in some schemas and also `"type": "integer"` is there a doc for the correct data types in the resource schema? I found one but that was for templates, wondering if it could be different for schemas

Reply:
Was able to find that within the CFN resource type schema (https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html), the schema used for `properties` is a subset of json schema draft 7. With this in mind, [json schema](https://json-schema.org/understanding-json-schema/reference/type#type-specific-keywords) defines both `integer` and `number` types with the latter used for representing both floats and integers. Hope this clarifies the doubt.